lib/checkers/zookeeper_health_checker.rb

```diff
@@ -66,6 +73,21 @@ def check
 
   private
 
+  def active_health_check(check_details)
+    node_path = zk_client.create("/#{hostname}", ephemeral: true, sequential: true)
```

Comment:
Should the test node be name spaced in `/whazzup/#{hostname}` to avoid conflicts if a hostname based root node is used for other purposes?